test/gui/shared/scripts/pageObjects/PublicLinkDialog.py

```diff
@@ -137,17 +159,45 @@ def setExpirationDate(self, context, publicLinkName, resource):
         squish.nativeType(expDate.day)
         squish.nativeType(expYear)
         squish.nativeType("<Return>")
+
+        actualDate = squish.waitForObjectExists(self.EXPIRATION_DATE_FIELD).displayText
+        expectedDate = f"{expDate.month}/{expDate.day}/{expYear}"
+        if not actualDate == expectedDate:
+            # retry with workaround
+            self.setExpirationDateWithWorkaround(expYear, expDate.month, expDate.day)
+
         squish.testSettings.silentVerifications = True
         squish.waitFor(
-            lambda: (test.xvp("publicLinkExpirationProgressIndicatorInvisible"))
+            lambda: (test.vp("publicLinkExpirationProgressIndicatorInvisible"))
```

Comment:
why did you do this? Is it not failing anymore? When did this happen?

Reply:
Yeah, I forgot about that.
what is that for actually? Before the test was failing locally so I had to change that one.
Haven't tried recently though. I will

Reply:
@swoichha any idea why and since when is it passing?

Reply:
I found out that `xvp` skips the test steps below so that the test scenario is shown as `pass`.
And with `vp` all the test steps will get to run and the test scenario is also passing. so good to have these changes.

Reply:
yeah, xvp is like `expected to fail`

Reply:
@saw-jan @Talank The reason we had `xvp` was due to this issue https://github.com/owncloud/client/issues/8733. As the checkbox use to be unchecked we did use `xvp` for that but now this issue has been fixed and closed. So, IMO replacing `xvp` with `vp` should make the test work.